virt/release_notes/virt-4-18-release-notes.adoc

```diff
@@ -65,11 +65,13 @@ This release adds new features and enhancements related to the following compone
 
 [id="virt-4-18-virtualization"]
 === Virtualization
+//CNV-48244: Instance types hotplug GA
+* You can now change the instance type associated with a running virtual machine (VM). This feature was previously Technology Preview and is now generally available.
```

Comment:
```suggestion
* You can now change the instance type associated with a running virtual machine (VM) and the change will take effect right away. This feature was previously Technology Preview and is now generally available.
```
You could also change the instance type before, the main change is that now you won't need to restart the VM for the changes to take effect.

Reply:
Thanks, @RoniKishner. Changed to "You can now change the instance type associated with a running virtual machine (VM) without restarting the VM." Is this ok?